### DISCUSSION TITLE
/v1/conversation/completion vs /v1/api/completion ?

### DISCUSSION THREAD

**Discussion by kaleocheng** [infiniflow/ragflow#2623] [Q&A]
I saw the webui is using /v1/conversation/completion to get answer but from the api doc, it's /v1/api/completion. is there  any difference between them or it's just an alias?

asking this as I'm getting  different responses from both api for the same query:
`/v1/conversation/completion`: it waited longer(4-10s ) for the response from knowlege RAG, so it retuned a correct answer eventually
`/v1/api/completion`: it  returned to client in 2-4 seconds with empty knowlege chunks, so I didn't get the proper answer eventually

**Comment by Feiue** [comment]
They are different APIs. But the responses are same. The chunks are just  not displayed in `/v1/api/completion`. You can refer to [api](https://github.com/infiniflow/ragflow/blob/main/docs/references/api.md)
